src/partials/home/hero.js

```diff
@@ -254,7 +254,9 @@ const Hero = ({
             </LinkItem>
           ))}
         </LinkContainer>
-        <LearnMore to="Features">LEARN MORE</LearnMore>
+        <LearnMore to="Features" smooth offset={-25} duration={500}>
```

Comment:
I wonder if we can extract these so that the scrolling is consistent across the app?